lib/RN2483-Arduino-Library/src/rn2xx3_handler.cpp

```diff
@@ -1236,6 +1244,13 @@ bool rn2xx3_handler::setTXoutputPower(int pwridx)
   return sendMacSet(F("pwridx"), String(pwridx));
 }
 
+void rn2xx3_handler::sendAutoBaud()
+{
+  _serial.write(static_cast<uint8_t>(0x00));
+  _serial.write(static_cast<uint8_t>(0x55));
```

Comment:
you should change baud to 9600 send 0x00, than to desired/default baud and send 0x55
but I see there are issues with wake up event on newer firmwares of RN2483

Reply:
Problem is we don't have a "serial" object there, only a stream.
A stream cannot set its baudrate.
But if a RN2483 is put to sleep (not the initial autobaud feature) it will still awake by sending a 0x55.
I just added the 0 and newline to make sure it is having a bit more time to read the 0x55 and also the newline to make sure that if it was in some sort of half read command state, it would flush that command (e.g. the CRLF was missed somehow)

So it is more like a "flush whatever you were waiting for, we start over"

Reply:
By the way, glad to know someone is actually reading my code too :)

Reply:
@TD-er so this method should be named wakeUp
did you tried to mesure minimum/maximum duration to start autonegotiation on your module maybe it is far too long?

Reply:
> 
> 
> By the way, glad to know someone is actually reading my code too :)

without reading how to learn to write?

Reply:
The datasheet describes you should pull the TX line down for longer than 8 bits + start/stop (10 - 11 bits time)
So writing a 0 byte at low baudrate can also help here.
For the normal GPIO pins (when using HW serial or SW serial) you can pull the pin down, but for the I2C => UART you cannot pull it down like a GPIO.
That's why I turn down the baudrate and write a 0 byte.

I will rename the function, as wake is more appropriate there.